docs/commands.md

```diff
@@ -185,8 +185,9 @@ usage: git elegant deliver-work
 ```
 
 Updates the current branch by rebasing the default upstream branch. Then,
-it pushes HEAD to appropriate upstream branch. The name of remote branch is
-equal to the local one.
+it pushes HEAD to appropriate upstream branch. By default, the name of remote
+branch is equal to the local one. Also, you can give a custom name of the remote branch if needed.
```

Comment:
Please split the line after 80 characters (https://github.com/bees-hive/elegant-git#architecture).